scripts/base/protocols/conn/main.zeek

```diff
@@ -281,6 +286,8 @@ function set_conn(c: connection, eoc: bool)
 		if ( c$history != "" )
 			c$conn$history=c$history;
 		}
+
+	c$conn$ip_proto = c$id$proto;
```

Comment:
`c$id$proto` uses 65535 as default, while `c$conn$ip_proto` is optional, so should this be protected via an `if ( c$id$proto != 65535 )`?